metadata.json

```diff
@@ -32,7 +32,9 @@
     {
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
-        "7"
+        "7",
+        "8",
+        "9"
```

Comment:
Can you call it 'Addd EL8/EL9 support' and also extend the CentOS version list?

Reply:
sure